app/controllers/all/opening/projects_controller.rb

```diff
@@ -23,7 +23,7 @@ def download_csv
 
     @csv = OpeningProjectsCsvExporter.new(@projects).call
 
-    send_data @csv, filename: "opening_#{month}_#{year}.csv", type: :csv, disposition: "attachment"
+    send_data "\uFEFF" + @csv, filename: "opening_#{month}_#{year}.csv", type: :csv, disposition: "attachment"
```

Comment:
Are we able to but this in a method in the `OpeningProjectsCsvExporter`? It feels a little icky here, it's so unclear what it is doing, wrapping it in a helpfully named method would help a lot.